conf/mesos-site.xml

```diff
@@ -74,7 +79,7 @@
 
   <property>
     <name>mesos.hdfs.executor.cpus</name>
-    <value>0.1</value>
+    <value>0.5</value>
```

Comment:
Well, it does launch and monitor HDFS. We will be adding health checks, error handling, etc. I felt a larger value was safer although this is of course configurable.